tracks/python/exercises/markdown/README.md

```diff
@@ -0,0 +1,90 @@
+### Problem and challenges
+	
+The markdown exercise asks the student to refactor some code. The original code is 
+hard to follow; it is constructed of a single `for` loop, has many nested 
+conditionals, and meaningless variable names. The goal is for them to produce 
+improved code that still passes the tests.
+	
+### Reasonable solution
+	
+```python
+import re
+
+
+def format_header(line):
+    def replacement(match):
+        hashes, text = match.groups()
+        num = len(hashes)
+        return f'<h{num}>{text}</h{num}>'
+    line = re.sub(r'^(#{1,6}) (.*)', replacement, line)
+    return line
+
+
+def format_bold(curr):
+    curr = re.sub(r'__(.*?)__', r'<strong>\1</strong>', curr)
+    return curr
+
+
+def format_italic(curr):
+    curr = re.sub(r'_(.*?)_', r'<em>\1</em>', curr)
+    return curr
+
+
+def format_list_item(line):
+    line = re.sub(r'^\* (.*)', r'<li>\1</li>', line)
+    return line
+
+
+def format_paragraph(line):
+    m = re.match('<h|<ul|<p|<li', line)
```

Comment:
The **m** is never used again ... `if re.match([...])` instead? Would save being yelled at for a single letter variable in pylint as well.